docs/operations/deploy_common_instancetypes.md

```diff
@@ -0,0 +1,29 @@
+# Deploy common-instancetypes
+
+The [`kubevirt/common-instancetypes`](https://github.com/kubevirt/common-instancetypes) provide a set of [instancetypes and preferences](../virtual_machines/instancetypes.md) to help create KubeVirt [`VirtualMachines`](http://kubevirt.io/api-reference/main/definitions.html#_v1alpha1_virtualmachine).
+
+Beginning with the 1.1 release of KubeVirt, cluster-wide resources can be deployed directly through KubeVirt, without the need to install another operator.
```

Comment:
You might need another sentence here to complete the thought, e.g.

"... cluster wide resources can be deployed directly... without another operator..."  This allows deployment of a set of default instance types and preferences along side KubeVirt.